src/components/AgentCard/AgentCard.stories.tsx

```diff
@@ -23,96 +31,91 @@ const initialLetters = (name: string) => {
 	return "";
 };
 
-export const AgentCardStory = () => (
-	<>
-		<div className="neo-nav">
-			<AgentCard
-				agentName={"Barbara Barberson"}
-				agentStatus={"connected"}
-				avatar={
-					<Avatar
-						variant="basic"
-						label={"Barbara Barberson"}
-						initials={initialLetters("Barbara Barberson")}
-					/>
-				}
-			/>
-		</div>
-		<br />
-		<div className="neo-nav">
-			<AgentCard
-				agentName={"Barbara Barberson"}
-				agentStatus={"ready"}
-				avatar={
-					<Avatar
-						variant="basic"
-						label={"Barbara Barberson"}
-						initials={initialLetters("Barbara Barberson")}
-					/>
-				}
-			/>
-		</div>
-		<br />
-		<div className="neo-nav">
-			<AgentCard
-				agentName={"Barbara Barberson"}
-				agentStatus={"not-ready"}
-				avatar={
-					<Avatar
-						variant="basic"
-						label={"Barbara Barberson"}
-						initials={initialLetters("Barbara Barberson")}
-					/>
-				}
-			/>
-		</div>
-		<br />
-		<div className="neo-nav">
-			<AgentCard
-				agentName={"Barbara Barberson"}
-				agentStatus={"ready"}
-				avatar={<Avatar variant="generic" label={"Barbara Barberson"} />}
-			/>
-		</div>
-		<br />
+export const AgentCardStory: Story = {
+	render: () => (
+		<>
+			<div className="neo-nav">
+				<AgentCard
+					agentName={"Barbara Barberson"}
```

Comment:
You are only updating the existing stories, so I know it's not you that did this, but I cannot tell you how much it annoys me when we compute a string. Since you're already in here, can you update all these computed strings to just strings? 
```suggestion
					agentName="Barbara Barberson"
```

Reply:
✅